scripts_src/global/gl_k_alcohl.ssl

```diff
@@ -70,34 +60,26 @@ procedure drink(variable who, variable item) begin
     if (global_var(GVAR_ALCOHOL_ADDICT) == 1) then return;
 
     // regular drinking increases chances
-    if ((game_time - last_drink_date) >= SPORADIC_ALCOHOL_USAGE_TIME) then begin
+    if ((game_time - last_drink_date) >= ONE_GAME_WEEK) then begin
```

Comment:
No, I didn't mean to remove time defines, just rename them like `TIME_ALCOHOL_USAGE_SPORADIC`.
Because that's what you usually expect in completion.